src/api/color.rs

```diff
@@ -226,6 +227,16 @@ pub struct ChromaticityPoint {
   pub y: u16,
 }
 
+#[wasm_bindgen]
+impl ChromaticityPoint {
+  /// Chromaticity coordinates as defined by CIE 1931, expressed as 0.16
+  /// fixed-point values.
+  #[wasm_bindgen(constructor)]
+  pub fn constructor(x: u16, y: u16) -> Self {
```

Comment:
please add a `#[cfg()]` to make so it is not added to the non-js API, why is that needed btw?

Reply:
So the js user can add a new `ChromaticityPoint` via:
```ts
let a = new ChromaticityPoint(0, 0);
```